### PR TITLE
closes #48 lister can create job

### LINKED ISSUE #48: Lister Can Create a Job
As a logged in lister,
When I visit my dashboard,
And I click on "Create Job",
Then I am redirected to a create job form with fields for:
- Title
- Category (dropdown)
- Description
- City
- State
- Zipcode
- Bidding Close Date
- Must Complete By Date
- Duration Estimate (dropdown)
  And when I complete all the fields,
  And then I click on "Create Job",
  I am redirected to the job show page,
  And I see the information for the new job.

### PR DESCRIPTION
**Pull request by ToniRib**
@jwperry did all the awesome work of having a user be able to create a job, and I put some fixes on at the end because the time zone was messed up and because I realized we were not validating the length of the job description.

### CODE REVIEW

File: app/controllers/user/jobs_controller.rb [hunk 1]
```diff
@@ -34,15 +54,47 @@ def update
...
+    bid_close_date = params[:job][:bidding_close_date]
+    param_hour = params[:bid_close]["time(4i)"]
+    minute = params[:bid_close]["time(5i)"]
+
+    DateTime.strptime("#{bid_close_date} #{param_hour}:#{minute}", "%Y-%m-%d %k:%M")
+  end
+
+  def construct_must_complete_by_date
+    complete_by_date = params[:job][:must_complete_by_date]
+    DateTime.strptime("#{complete_by_date}", "%Y-%m-%d")
```
Comment (ToniRib) [inline app/controllers/user/jobs_controller.rb:87]: I actually disagree with this one, I think it's easier to read this way.

**Comment by ToniRib** [comment]
@jwperry @danjwinter this is ready for your review!

**Comment by jwperry** [comment]
Awesome job on the time zone fix, thanks!

**Comment by ToniRib** [comment]
Master is now merged in!

**Pull request merged**